src/testing.rs

```diff
@@ -150,15 +161,9 @@ impl Git {
             // messages. Usually, we can set this with `git commit -m`, but we have
             // no such option for things such as `git rebase`, which may call `git
             // commit` later as a part of their execution.
-            ("GIT_EDITOR", {
-                // Some systems have `/bin/true` and others have
-                // `/usr/bin/true`. Pick whichever one we can find.
-                if Path::new("/bin/true").exists() {
-                    "/bin/true"
-                } else {
-                    "/usr/bin/true"
```

Comment:
Thanks, this also fixes tests on Linux distros where `/usr/bin` and `/bin` do not exist.